setup.py

```diff
@@ -12,9 +12,8 @@ def read(fname):
 setup(
     name='loggo',
     version='3.0.1',  # DO NOT EDIT THIS LINE MANUALLY. LET bump2version UTILITY DO IT
-    author='Danny McDonald',
-    author_email='daniel.mcdonald@bitpanda.com',
-    description=('Python logging tools'),
+    author='Bitpanda GmbH',
```

Comment:
:|

Reply:
You against this change? Dont think the email address will be responding too much to anyone :)

Reply:
haha, it's fine, I'm still in LICENSE :D